src/api/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_02_05_113812) do
+ActiveRecord::Schema.define(version: 2021_06_03_113812) do
```

Comment:
This version number is off. How did you create the migration included in this PR? You edited its filename by hand, right? Because this timestamp is in the future, so it cannot be possible that you generated it with `bundle exec rails g migration MyMigration123`.